nixos/modules/programs/mininet.nix

```diff
@@ -0,0 +1,40 @@
+# Global configuration for mininet
+# kernel must have NETNS/VETH/SCHED
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg  = config.programs.mininet;
+
+  generatedPath = with pkgs; makeSearchPath "bin"  [
+    iperf ethtool iproute socat
+  ];
+
+  pyEnv = pkgs.python.withPackages(ps: [ ps.mininet-python ]);
+
+  mnexecWrapped = pkgs.runCommand "mnexec-wrapper"
+    { buildInputs = [ pkgs.makeWrapper pkgs.pythonPackages.wrapPython ]; }
+    ''
+      makeWrapper ${pkgs.mininet}/bin/mnexec \
+        $out/bin/mnexec \
+        --prefix PATH : "${generatedPath}"
+
+      makeWrapper ${pyEnv}/bin/mn $out/bin/mn
+
+      # mn errors out without a telnet binary
+      # pkgs.telnet brings an undesired ifconfig into PATH see #43105
+      # so we wrap it here instead
+      makeWrapper ${pkgs.telnet}/bin/telnet $out/bin/telnet
```

Comment:
```
ln -s ${pkgs.telnet}/bin/telnet $out/bin/telnet
```

Reply:
Why was this marked as resolved?

Reply:
Did you get a notification on resolution ? I marked it resolved (maybe too soon) because I fixed it locally.